free-1.md

```diff
@@ -57,12 +107,18 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I don't think there will be a lot of language decisions -- the input I need only requires 4 basic pieces of information from the user, probably in the form of some combination of text boxes / drop-down menus / check boxes.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
-
+I don't think this idea is too ambitious -- it's very easy to generate the names of the notes in a chord given the root, type of chord, and inversion. The tricky part would be to translate that into a visual representation of the chord. Maybe I could have a staff with ovals that represent each note, and after I generate the names and order of the notes in the chord, I could just enable those notes to be "visible" on the staff, and allow the user to download an image of the generated chord on the staff. 
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+I think this project really fits the bill of being a domain specific language, and will hopefully provide a helpful abstraction for the process of actually writing out each of the notes of a chord on a musical staff. 
+
+This might not be a good idea because it involves writing notes on a staff, which may require my language to communicate with an existing composition software. I can understand how to generate all the notes of a chord (given a root and key signature of a chord, I have a good idea about how to generate the names of the chords because the relationship between the notes in the chord is just dictated by their interval). I think a challenge would be to find a way to visually display the chord on the staff instead of just naming the notes in the chord.
+
```

Comment:
I agree that it would be easier to leverage an existing software to actually generate the images.  I know there are existing DSLs that generate musical scores, such as [LilyPond](http://lilypond.org/).  One idea would be to compile your DSL into LilyPond code, and then have them do all the hard work of displaying the notes.